cvat-cli/src/cvat_cli/cli.py

```diff
@@ -66,7 +65,7 @@ def tasks_create(
 
     def tasks_delete(self, task_ids: Sequence[int]) -> None:
         """Delete a list of tasks, ignoring those which don't exist."""
-        self.client.delete_tasks(task_ids=task_ids)
+        self.client.tasks.remove_by_ids(task_ids=task_ids)
```

Comment:
@zhiltsov-max , I don't know all use cases, but probably tasks.delete(task_ids) is better. First argument can be array of ids or array of tasks. What do you think?

Reply:
Its almost just a copy-paste from what CLI had. I can't say I prefer one way over another. Both work well and both are convertible to each other. I can say that your variant is more focused on manual typing, while the other is better for passing a container without the need of unpacking. I'm not sure if `delete(id1, id2, id3)` is more frequent pattern in the client code than `delete(ids)`.